src/back/ServicesFile.ts

```diff
@@ -37,11 +37,15 @@ export namespace ServicesFile {
       input: data,
       onError: onError && (e => { onError(`Error while parsing Services: ${e.toString()}`); })
     });
-    parsed.redirector = parseBackProcessInfo(parser.prop('redirector'));
-    parsed.fiddler    = parseBackProcessInfo(parser.prop('fiddler'));
-    parsed.server     = parseBackProcessInfo(parser.prop('server'));
-    parser.prop('start').array(item => parsed.start.push(parseBackProcessInfo(item)));
-    parser.prop('stop').array(item  => parsed.stop.push(parseBackProcessInfo(item)));
+
+    parsed.server = parseBackProcessInfo(parser.prop('server'));
+
+    if (process.platform !== 'darwin') {
```

Comment:
It looks like I forgot to flag these properties as optional. If you pass a second parameter to ``parser.prop(...)`` with the value of ``true`` then it will not log any errors if it is missing

Reply:
Are they not required on Windows or Linux?

Reply:
They should be optional on all platforms. These files are usually managed by other members of the staff, and we want it to be flexible

Reply:
I see, thanks.